src/test/regress/sql/full_join.sql

```diff
@@ -0,0 +1,91 @@
+--
+-- Full join with subquery pushdown support 
+--
+
+SET citus.next_shard_id TO 9000000;
+
+CREATE TABLE test_table_1(id int, val1 int);
```

Comment:
Minor: Given that you're using a generic name for the tables, it might make sense to create them in a separate schema.